yahoo_fantasy_api/team.py

```diff
@@ -113,28 +113,29 @@ def _compact_eligible_pos(j):
             pass
         return roster
 
-    def change_positions(self, day, modified_lineup):
+    def change_positions(self, modified_lineup, day=None, week=None):
         """Change the starting position of a subset of players in your lineup
 
         This raises a RuntimeError if any error occurs when communicating with
         Yahoo!
-
-        :param day: The day that the new positions take affect.  This should be
```

Comment:
We should probably rename `day` if we intend to treat it as a day or week.

Reply:
Renamed as time_frame, let me know if you have any other suggestions